templates/compose/docker-compose.yml.erb

```diff
@@ -18,6 +18,7 @@ services:
     image: <%= scope['nexus::image'] %>:<%= scope['nexus::version'] %>
     expose:
       - 8081
+      - <%= scope['nexus::smart_proxy_port'] %>
```

Comment:
Ok I updated it.
I declare ports as string to follow Docker best practices:

> Note: When mapping ports in the HOST:CONTAINER format, you may experience erroneous results when using a container port lower than 60, because YAML will parse numbers in the format xx:yy as sexagesimal (base 60). For this reason, we recommend always explicitly specifying your port mappings as strings.